user-guide/Advanced_Modules/User Defined APIs/UD_APIs_UserDefinableApiEndpoint.md

```diff
@@ -164,9 +169,33 @@ For example, this is the default configuration:
   ...
   "UserDefinableAPIs": {
     "NatsSubject": "Skyline.DataMiner.Protobuf.Apps.UserDefinableApis.Api.v1.UserDefinableApiTriggerRequest",
-    "SessionConfigPath": "C:\\Skyline DataMiner\\SLCloud.xml",
     "CredentialsConfigPath": "C:\\Skyline DataMiner\\NATS\\nsc\\.nkeys\\creds\\DataMinerOperator\\DataMinerAccount\\DataMinerUser.creds",
-    "MessageBrokerTimeOutSeconds": 300
+    "MessageBrokerTimeOutSeconds": 90
   }
 }
 ```
+
+#### Changing the time-out
+
+IIS has a time-out set to 120 seconds (2 minutes). When increasing the **MessageBrokerTimeOutSeconds** as explained above, this time-out in IIS will also need to be increased.
+
+> [!NOTE]
+> Make sure there is a margin, put the time-out in IIS higher than than the **MessageBrokerTimeOutSeconds**.
+
+You can change the time-out in IIS with following steps:
+
+1. Open `Internet Information Services (IIS) Manager`.
+
+1. In the *Connections* pane on the left, select the top icon with the server name and click *Application Request Routing*.
```

Comment:
I seem to be missing Application Request Routing in my IIS manager 😕

Reply:
Could it be I need to install something? <https://techcommunity.microsoft.com/t5/iis-support-blog/application-request-routing-is-missing-in-iis-manager/ba-p/2238317>

Reply:
My bad. It's not available when opening the IIS Manager in Computer Management, see [screenshot](https://skylinebe-my.sharepoint.com/:i:/g/personal/peter_vanpoucke_skyline_be/EeCbOoIOJtxHtPWdFyhM5LUBufuQpCt0SX9noPb3CDvtSA?e=34TJAV).

Reply:
I changed the name to be correct *Application Request Routing Cache*. The steps mention to open the IIS manager and not computer managment, so I guess it's fine to keep it like this. It's weird that it's the only module not available when opening through Computer Management